### PR TITLE
fix(DB/SAI) Mobs regarding Planting the Banner

### PR DESCRIPTION
**Pull request by v-mstrs**
## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/2996
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/15018
- Closes https://github.com/chromiecraft/chromiecraft/issues/4981

## Tests Performed:

- Tested

## How to Test the Changes:

1. .go xyz 1703.4 6329.51 24.73 530
2. .quest add 10518, .additem 30416
3. place banner

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by heyitsbench** [comment]
Any source on those waypoint targets?

**Comment by v-mstrs** [comment]
Nope, just roughly

**Comment by Gultask** [comment]
Yeah sorry we can't accept this since we only take sniffed waypoints (from official servers) or waypoints used by other cores such as Mangos or TC, since they often have the same standards.

File: data/sql/updates/pending_db_world/rev_1682739029073809022.sql [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 9 AND `entryorguid` = 1995200);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(1995200, 9, 0, 0, 0, 0, 100, 0, 0, 0, 0, 0, 0, 69, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1732.26, 6328.56, 11.405, 2.947, ''),
+(1995200, 9, 1, 0, 0, 0, 100, 0, 0, 0, 0, 0, 0, 69, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1703.16, 6333.94, 23.708, 4.298, ''),
+(1995200, 9, 2, 0, 0, 0, 100, 0, 0, 0, 0, 0, 0, 69, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1690.59, 6321.71, 31.229, 5.377, ''),
+(1995200, 9, 3, 0, 0, 0, 100, 0, 0, 0, 0, 0, 0, 69, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1695.82, 6315.29, 32.181, 5.386, '');
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1682739029073809022.sql:12]: Is this supposed to be a path or random points which they move to?
If it's the former you use the `waypoints` table, the latter you need to use distinct entries and then call them from action 87 or 88.

Also comments

**Comment by Gultask** [comment]
@v-mstrs Ty for the effort nonetheless!
If you want feel free to join the AC Discord, we can help you out with this PR when we're available!

**Comment by nyilas7** [comment]
> Any source on those waypoint targets?

I've tested, this quest works in CMaNGOS, but does not work with TrinityCore.

Not yet found the waypoint targets in CMaNGOS DB.

**Pull request closed without merging**